<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1250</width>
    <height>784</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1201</width>
      <height>701</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="tijera">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>730</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Tijera</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>730</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Cantidad:</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="cantidadTijeras">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>730</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="piedra">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>730</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Piedra</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>730</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Cantidad:</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="cantidadPiedras">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>730</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="papel">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>730</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Papel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>730</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Cantidad: </string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="cantidadPapeles">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>730</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
